### PR TITLE
Create workloads from Templates

### PR DESCRIPTION
**Pull request by gajanan-more**
**Fix:** https://issues.redhat.com/browse/ODC-5552

**Problem:** Automation to create workloads using different types of Templates

**Solution:** This script will create workloads using different types of Templates from Developer Catalog

**Test Setup:**
1. Run `yarn run test-cypress-devconsole` in `frontend`
2. Execute feature file: `frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature`

**Screenshots:**
Screenshot from 2021-02-25 16-05-30

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature [hunk 1]
```diff
@@ -1,54 +1,36 @@
...
              Then user will be redirected to Topology page
-              And user is able to see workload "nginx-example" in topology page
+              And user is able to see workload "<workload_name>" in topology page
+
+        Examples:
+                  | template_type | card_name                             | workload_name             |
+                  | Databases     | MariaDB                               | mariadb                   |
+                  | Languages     | Node.js + PostgreSQL (Ephemeral)      | nodejs-postgresql-example |
+                  | Middleware    | Apache HTTP Server                    | httpd-example             |
+                  | Other         | Nginx HTTP server and a reverse proxy | nginx-example             |
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature:36]: I think we can move this scenario to create-from-catalog.feature

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature [hunk 2]
```diff
@@ -1,54 +1,36 @@
...
-             When user selects CI/CD type
-              And user searches and selects Template card "Jenkins" from catalog page
-              And user clicks on Instantiate Template button
-              And user clicks on Create button
+        @smoke
+        Scenario: Deploy Application using CI/CD type of Catalog Templates
+             When user selects "CI/CD" from Templates type
+              And user searches and selects Template card Jenkins from catalog page
+              And user clicks Instantiate Template button on side bar
+              And user clicks create button
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature:17]: We can more this scenarios to create-from-catalog.feature

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature (hunk 1 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature]: Updated

File: frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature (hunk 2 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/addFlow/create-from-templates.feature]: Updated

**Pull request merged**